pkg/config/hardfork.go

```diff
@@ -17,12 +17,15 @@ const (
 	HFBasilisk // Basilisk
 )
 
+// Hardforks represents the ordered slice of all possible hardforks.
+var Hardforks = []Hardfork{HFAspidochelone, HFBasilisk}
```

Comment:
Can we avoid repeating them?

Reply:
We can, but we need an ordered set of hardforks for validation checks simplicity. I've kept this list and removed the map, is it better?

Reply:
Please keep the map. What I mean is iterating over HF constants (from zero to the last+1 one) and filling this array in without copying names. So you wouldn't need to change this code once C one comes in.

Reply:
Good, fixed.

Reply:
However, we still to update the right bound in `for` loop each time new hardfork is added.

Reply:
You can add unexprorted `hf_last` and do `i < hf_last`

Reply:
See the updated code, like that?